vaadin-number-field.html

```diff
@@ -0,0 +1,192 @@
+<!--
+@license
+Copyright (c) 2017 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../polymer/lib/mixins/gesture-event-listeners.html">
+<link rel="import" href="../vaadin-themable-mixin/vaadin-themable-mixin.html">
+<link rel="import" href="vaadin-text-field.html">
+
+<dom-module id="vaadin-number-field-default-theme">
+</dom-module>
+
+<dom-module id="vaadin-number-field-template">
+  <template>
+    <style>
+      :host {
+        width: 120px;
+      }
+
+      :host([has-controls]) [part="value"] {
+        text-align: center;
+      }
+
+      /* Value control styles */
+      [part="decrease-button"],
+      [part="increase-button"] {
+        display: inline-block;
+        cursor: pointer;
+        padding: 0 8px;
+        opacity: 0.5;
+      }
+
+      [part="decrease-button"][hidden],
+      [part="increase-button"][hidden] {
+        display: none;
+      }
+
+      [part="decrease-button"]::before {
+        content: "–"; /* en dash */
+      }
+
+      [part="increase-button"]::before {
+        content: "+";
+      }
+
+      /* Hide the native arrow icons */
+      [part="value"]::-webkit-outer-spin-button,
+      [part="value"]::-webkit-inner-spin-button {
+        -webkit-appearance: none;
+        margin: 0;
+      }
+
+      [part="value"] {
+        -moz-appearance: textfield;
+      }
+    </style>
+
+    <div 
+      part="decrease-button" 
+      on-tap="_decreaseValue" 
```

Comment:
Why change this? Switching from tap to click makes clicking the buttons laggy on iOS